distributed/tests/test_nanny.py

```diff
@@ -13,11 +13,6 @@
 
 import psutil
 import pytest
-
-from distributed.diagnostics.plugin import WorkerPlugin
-
-pytestmark = pytest.mark.gpu
```

Comment:
this pytestmark was ignored because the later `pytestmark = pytest.mark.ci1` overrode it

Perhaps `test_nanny.py` should not be marked gpu and this line should be removed rather than fixed?